installer/scripts/agent_topology_request.rb

```diff
@@ -0,0 +1,126 @@
+require 'optparse'
+require 'gyoku'
```

Comment:
No, it isn't I will have to add it to the gemlist. It has a dependency called Builder, both are MIT licensed.